packages/flutter/lib/src/material/divider.dart

```diff
@@ -26,19 +26,22 @@ import 'theme.dart';
 class Divider extends StatelessWidget {
   /// Creates a material design divider.
   ///
-  /// The height must be at least 1.0 logical pixels.
+  /// The height must be positive.
   const Divider({
     Key key,
     this.height: 16.0,
     this.indent: 0.0,
     this.color
-  }) : assert(height >= 1.0),
+  }) : assert(height >= 0.0),
        super(key: key);
 
   /// The divider's vertical extent.
   ///
-  /// The divider itself is always drawn as one logical pixel thick horizontal
+  /// The divider itself is always drawn as one device pixel thick horizontal
   /// line that is centered within the height specified by this value.
+  ///
+  /// Divider of 0.0 height is drawn as one device pixel line without any
```

Comment:
Very minor but just curious: when I was working on a PR for the DartSDK they had a convention to not use future tense in documentation (in particular this [comment](https://github.com/dart-lang/sdk/pull/26012#r75671896)). Is it applicable to Flutter?

Would "A divider with a height of 0.0 is always drawn as a line with a height of exactly one device pixel, without any padding around it." work?